accounting/test/server/api.data.ts

```diff
@@ -1,10 +1,11 @@
 import { Scope } from "src/server/auth"
+import { CreditCommonsTransaction } from "../../src/model/creditCommons"
 
 export const testCurrency = (props?: any) => {
   props = {
     ...props,
     settings: {
-      defaultInitialCreditLimit: 1000,
+      defaultInitialCreditLimit: 100000,
```

Comment:
This makes quite a few existing tests to fail. I agree that it is a more reasonable value but you can just use the props parameter and overwrite the default 1000 by 100000.